tests/checks/tmux-abbr.fish

```diff
@@ -1,13 +1,12 @@
 #RUN: %fish %s
 #REQUIRES: command -v tmux
 
-set -g isolated_tmux_fish_extra_args -C '
+isolated-tmux-start -C '
```

Comment:
yeah there's a lot that could be improved about these tests,
we should probably find a shorter name for "isolated-tmux",
probably we should simply call it "tmux".

(we probably don't strictly need a new tmux client process everytime,
we could open a longer lived tmux in control mode and echo commands
to its stdin (via a fifo).. but I'm not sure if that's an easy change)

Reply:
we should maybe also teach our `tmux send-keys` wrapper to accept our `ctrl-g` notation